OWNERS_ALIASES

```diff
@@ -11,34 +11,25 @@ aliases:
   - strongjz
 
   ingress-nginx-maintainers:
-  - rikatz
-  - strongjz
   - cpanato
+  - Gacko
   - puerco
   - tao12345666333
 
   ingress-nginx-reviewers:
-  - rikatz
```

Comment:
should keep as well

Reply:
See above.

Reply:
Sorted stuff in recent update, so gonna resolve that one.